autometacal/python/metacal.py

```diff
@@ -65,29 +82,95 @@ def generate_mcal_image(gal_image,
   # Compute inverse Fourier transform
   img = tf.math.real(tf.signal.fftshift(im_reconv))
 
-  return img
+  return img[:,fact*nx:-fact*nx,fact*ny:-fact*ny]
```

Comment:
you can use tf.image.resize_with_crop_or_pad for that as well Ii think

Reply:
Yes, but this works...

Reply:
sure